modules/cert-manager-acme-about.adoc

```diff
@@ -7,3 +7,8 @@
 = About ACME issuers
 
 The ACME issuer type for the {cert-manager-operator} represents an Automated Certificate Management Environment (ACME) certificate authority (CA) server. ACME CA servers rely on a _challenge_ to verify that a client owns the domain names that the certificate is being requested for. If the challenge is successful, the {cert-manager-operator} can issue the certificate. If the challenge fails, the {cert-manager-operator} does not issue the certificate.
+
+[NOTE]
+====
+Private DNS zones are not supported with Let’s Encrypt and internet ACME servers.
```

Comment:
Not going to hold up merge over it, but I noticed this and another spot below didn't have _Let's Encrypt_ italicized, but all the other instances were. Possibly worth a follow-up PR if they should synced up for consistency.